.github/ISSUE_TEMPLATE/s2_report.yaml

```
@@ -0,0 +1,98 @@
+name: Spectrum 2 Bug Report
+description: Report an issue with the Spectrum 2 theme in SWC 1.0
+title: '[S2 Bug]: '
+labels: [bug, triage, needs jira ticket, Spectrum 2]
+# assignees:
+body:
+    - type: markdown
+      attributes:
+          value: |
+              Thanks for taking the time to fill out this bug report! Your issue will be triaged within a week of filing the issue.
+    - type: checkboxes
+      id: terms
+      attributes:
+          label: Code of conduct
+          description: By submitting this issue, you agree to follow our [code of conduct](https://github.com/adobe/spectrum-web-components/blob/main/CODE_OF_CONDUCT.md#adobe-code-of-conduct).
+          options:
+              - label: I agree to follow this project's code of conduct.
+                required: true
+    - type: checkboxes
+      id: reviewed-issues
+      attributes:
+          label: Reviewed open issues
+          description: Before submitting this issue, please confirm that you have looked through the [existing Spectrum 2 issues](https://github.com/adobe/spectrum-web-components/labels/Spectrum%202) and confirmed that this is not a duplicate ticket.
+          options:
+              - label: I have confirmed that this is a new issue.
+                required: true
```

What do you think about making this a dropdown, so it is a bit more playful, something along the lines of:

```suggestion
    - type: dropdown
  id: reviewed-issues
  attributes:
      label: Have you reviewed existing [existing Spectrum 2 issues](https://github.com/adobe/spectrum-web-components/labels/Spectrum%202)?
      description: Please confirm if you've reviewed existing issues to avoid duplicates.
      options:
          - Yes, I’ve reviewed existing issues.
          - Oops, I’ll check now.
```